src/homework/day8/collections/MyFigures.java

```diff
@@ -0,0 +1,55 @@
+package homework.day8.collections;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class MyFigures {
+
+    public static void main(String[] args) {
+
+        List<String> myFiguresList = Arrays.asList("Oval", "Rectangle", "Circle", "Square", "Ellipse");
+
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter("C:\\Users\\Tatsiana_Skorabahata\\Desktop\\automation\\JavaCourse\\files\\collectionsTest\\figures.txt"));
+            String stringToWrite = "";
+            for (String figure : myFiguresList) {
+
+                if (myFiguresList.indexOf(figure) != myFiguresList.size() - 1) {
+                    stringToWrite += figure + "-";
+                } else {
+                    stringToWrite += figure;
+                }
+
+            }
+            writer.write(stringToWrite);
+            writer.close();
+        } catch (IOException e) {
+
```

Comment:
Extra new line. Remove. The same for such cases.

Reply:
Extra line is removed.